### DISCUSSION TITLE
[Translation] Missing translation (unable to find it in Weblate)

### DISCUSSION THREAD

**Discussion by ldmpub** [kimai/kimai#4982] [General] [answered]
Hi,

**Can you please help me find the following string in Weblate?**
_(I'm trying to translate it to french but ... I can not find it in Weblate! Can it be "harder coded" in Kimai?_ 😮)

**Illustration**
_(it's the only main menu item not displayed in french when Kimai language is set to FR)_
![image](https://github.com/user-attachments/assets/6c159619-d1d6-4ee5-acf1-184d2616a13d)

**String searches I performed in Weblate**

- Here is the query to search the source string (in all Kimai projects): https://hosted.weblate.org/search/kimai/?q=+source%3A"Hours+approval"
- Here is the query to search in the translated strings only (still for all Kimai projects and for all translated languages): https://hosted.weblate.org/search/kimai/?q=target%3A%22Hours+approval%22

=> Both queries results are Empty.

Waiting for your answers to help me improve Kimai's french translation 🤞,

**Comment by kevinpapst** [comment, marked as answer]
Not available in Weblate, as this is coming from an extension.

**Reply by ldmpub** [reply]
Thanks it's clear 👍.

Can you please give me the **name of this extension?** _(I'm not the admin of this on-premise Kimai instance)_ 

And, if you know it by heart💛, where this extension manage its translation? otherwise I will digg by myself base on the given name 😅.

**Reply by ldmpub** [reply]
Seems to be: https://www.kimai.org/fr/store/katjaglass-approval-bundle.html, am I right?

**Reply by ldmpub** [reply]
Ping @KatjaGlassConsulting:

Is your extension for Kimai 👆 meant to be translatable? _(without too many technical knowledge of xlf / XLIFF format!)_

Can you add a french one? (before processing, please tell me the number of strings and words 😋 it contains).

PS: see 3 translations yet avaible for this Kimai extension => [katjaglass-approval-bundle.html](https://github.com/KatjaGlassConsulting/ApprovalBundle/tree/main/Resources/translations)

**Comment by ldmpub** [comment]
This Discussion is answered => I close it ✔.

Plugin translation is possible => I have opened an issue in the plugin repo, see:  
* https://github.com/KatjaGlassConsulting/ApprovalBundle/issues/52 <br><br>

@KatjaGlassConsulting 😉
